holopy/core/metadata.py

```diff
@@ -233,3 +244,21 @@ def theta_phi(self, origin=None):
             for j, phi in enumerate(self.phi):
                 pos[i*self.shape[1]+j] = theta, phi
         return pos
+
+def interpret_args(schema=None, index=None, wavelen=None, polarization=None, optics=None):    
+    from .marray import Schema
+    if not isinstance(schema, Schema):
+        schema = Schema(positions=schema)
```

Comment:
Will this behave correctly if you are loading an image? The schema positions argument is meant to be a list of coordinates, here I think you can end up setting it equal to a list of pixel values.